cloudevents/formats/cbor-format.md

```diff
@@ -0,0 +1,199 @@
+# Avro Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The CBOR Format for CloudEvents defines how events attributes are expressed in
+the [CBOR Specification][cbor-spec].
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Attributes](#2-attributes)
+3. [Envelope](#3-envelope)
+
+## 1. Introduction
+
+[CloudEvents][ce] is a standardized and protocol-agnostic definition of the
+structure and metadata description of events. This specification defines how the
+elements defined in the CloudEvents specification are to be represented in the
+Concise Binary Object Representation (CBOR) Data Interchange Format ([RFC 8949][cbor-spec]).
+
+The [Attributes](#2-attributes) section describes the naming conventions and
+data type mappings for CloudEvents attributes.
+
+The [Envelope](#3-envelope) section defines a CBOR container for CloudEvents
+attributes and an associated media type.
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][rfc2119].
+
+## 2. Attributes
+
+This section defines how CloudEvents attributes are mapped to CBOR. This
+specification does not explicitly map each attribute, but provides a generic
+mapping model that applies to all current and future CloudEvents attributes,
+including extensions.
+
+For clarity, extension attributes are encoded using the same rules as
+core attributes. This includes their syntax and placement
+within the CBOR data item. In particular, extensions are placed as top-level CBOR
+key-value pairs of a map. Extensions MUST be encoded as a top-level CBOR key-value pairs.
+
+### 2.1. Base Type System
+
+The core [CloudEvents specification][ce] defines a minimal abstract type system,
+which this mapping leans on.
+
+### 2.2. Type System Mapping
+
+The [CloudEvents type system][ce-types] MUST be mapped to CBOR types as follows,
+with exceptions noted below.
+
+| CloudEvents   | CBOR                                                         |
+| ------------- | ------------------------------------------------------------ |
+| Boolean       | [Simple value][cbor-fpnocont] number `20` for `false` and number `21` for `true` |
```

Comment:
I would add an explicit link with a reference to the CBOR RFC section number 3.3 in the text here since this is instantly confusing to people who haven't looked at CBOR. The normative statement here should be that the CBOR convention for encoding true or false is used, 

e.g. "CBOR simple value 'true' (21) or CBOR simple value 'false' (20)"